src/test/java/org/apache/commons/collections4/iterators/CollatingIteratorTest.java

```diff
@@ -16,12 +16,14 @@
  */
 package org.apache.commons.collections4.iterators;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Comparator;
-import java.util.List;
-
 import org.apache.commons.collections4.comparators.ComparableComparator;
+import org.junit.Test;
+
+import java.lang.reflect.Array;
+import java.util.*;
```

Comment:
Thanks for your advice!
However, in general I've turned that feature in Intellij off.

However again, I often reorganize imports manually to make the code more compact.
And in the end often select the code developed by me and reformat it.

Honestly, couldn't simply imagine that this is such a BIG THING in Open Source projects ...
At first I thought some are joking or trying to play tricks on me ...

Thanks for turning it back in this commit.
Furthermore, if you want me to recheck this commit regarding this I'll do and push the changes.

And I vow to try to keep this in mind and try to improve regarding this in Open Source projects. :-)